### PR TITLE
Summer Ball changes

### PR DESCRIPTION
**Pull request by nervere**
## Changelog
:cl:
add: Summer ball beach map! Many thanks to Jerry Derpington, MMMiracles, BeeSting12, CAPTTLasky, Nalzul, Niknak, and Mrfluff for their contributions to this!
add: Headpatting
add: Wearable pet collars
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
^w^

**Comment by Ty-the-Smonk** [comment]
Now THIS is a cursed pr

**Comment by AcapitalA** [comment]
> add: Wearable pet collars

So…     why? I don't like where this is going.

**Comment by Rowelll** [comment]
> > add: Wearable pet collars
> 
> So… why? I don't like where this is going.

G A M E R
M O M E N T

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -253,6 +253,9 @@
 			return
 		M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
 						"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
+	else if(check_zone(M.zone_selected) == "head") //Headpats!
+		M.visible_message("<span class='notice'>[M] gives [src] a pat on the head to make [p_them()] feel better!</span>", \
```
Comment (wesoda25) [inline code/modules/mob/living/carbon/carbon_defense.dm]: M.visible_message("<span class='notice'>[M] punches [src] in the eye to make [p_them()] feel better!</span>", \
"<span class='notice'>You punch [src] in the eye to make [p_them()] feel better!</span>")

File: code/modules/clothing/neck/_neck.dm [hunk 2]
```diff
@@ -191,18 +191,13 @@
...
+	desc = "It's for pets, and cute catgirls."
 	icon_state = "petcollar"
 	item_color = "petcollar"
 	var/tagname = null
 
-/obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
-		return FALSE
-	return ..()
-
```
Comment (wesoda25) [inline code/modules/clothing/neck/_neck.dm]: /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
	if(ishuman(M))
		return FALSE
	return ..()

Comment (wesoda25) [reply code/modules/clothing/neck/_neck.dm]: would work better here

**Comment by Whoneedspacee** [comment]
can you not be a hypocrite with this cat bullshit, christ.

**Comment by 81Denton** [comment]
Good job on the map!

Btw, no need to argue about c*t content since this is staying test merge only.

**Comment by 81Denton** [comment]
Also, please check the Travis output since this has several runtimes and the linter is complaining.

**Comment by nervere** [comment]
Summer Ball is over, so I'm going to close this now.
Thank you all for attending, and thanks again for the hard work of all the mapmakers and admins!
See you all at the Winter ball!! :-)

**Pull request closed without merging**